virtManager/asyncjob.py

```diff
@@ -76,6 +76,16 @@ def cb_wrapper(callback, asyncjob, *args, **kwargs):
             asyncjob.job_canceled):
             return  # pragma: no cover
 
+        # If network is inactive, show a different error message
+        if (isinstance(e, libvirt.libvirtError) and
+                e.get_error_code() == libvirt.VIR_ERR_OPERATION_INVALID and
+                e.get_error_domain() == libvirt.VIR_FROM_NETWORK):
+            asyncjob.set_error(
+                str(e) + "\n\n" +
+                _("Go to: Edit -> Connection Details -> Network Interfaces to activate the network."),
+                "".join(traceback.format_exc()))
+            return # pragma: no cover
+
```

Comment:
This works but it's not correct way to do it. getting error_domain `VIR_FROM_NETWORK` and error_code `VIR_ERR_OPERATION_INVALID` doesn't guarantee that it is due to not-activated network.